helm/templates/_resources.tpl

```diff
@@ -0,0 +1,6 @@
+{{/*
+Create configmap name as used by the service name label.
+*/}}
+{{- define "configmap.fullname" -}}
+{{- printf "%s-%s-%s" .Release.Name .Chart.Name "configmap" | indent 1 }}
+{{- end }}
```

Comment:
please check that all files end ther is not EOF warning

Reply:
done